core/client_test.go

```diff
@@ -10,23 +10,20 @@ import (
 )
 
 func TestRemoteClient_Status(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
-	t.Cleanup(cancel)
-
 	client := StartTestNode(t).Client
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*4)
+	t.Cleanup(cancel)
 	status, err := client.Status(ctx)
 	require.NoError(t, err)
 	require.NotNil(t, status)
 }
 
 func TestRemoteClient_StartBlockSubscription_And_GetBlock(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
-	t.Cleanup(cancel)
-
 	client := StartTestNode(t).Client
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*4)
+	t.Cleanup(cancel)
```

Comment:
What's the reason for these ctx changes? We have the unspoken convention in the node that ctx is created at the top of the test.

Reply:
we can create the ctx at the top, but creating after results in a more consistent timing since we don't have to wait for the network to boot up

Reply:
Usually we set the timeout for the whole test, but as long StartTestNode avoids ctx, we can start timeout after the network start